token-metadata/program/tests/escrow.rs

```diff
@@ -0,0 +1,213 @@
+#![cfg(feature = "test-bpf")]
+pub mod utils;
+
+use solana_program::borsh::try_from_slice_unchecked;
+use solana_program_test::*;
+use solana_sdk::{signer::Signer, transaction::Transaction};
+use utils::*;
+
+mod escrow {
+    use mpl_token_metadata::{escrow::find_escrow_account, state::EscrowAuthority};
+    use solana_program::program_pack::Pack;
+
+    use super::*;
+
+    #[tokio::test]
+    async fn smoke_test_success() {
```

Comment:
Is there a linear item to create more tests that inject more failure scenarios?  If not should there be?

Reply:
No but I'll make one.